draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -1129,6 +1400,150 @@ After the validation, the Verifier will have the following data for further proc
 
 <{{examples/03-pid/verified_contents.json}}
 
+## Example 2: Type Metadata {#ExampleTypeMetadata}
+
+```json
+{
+  "vct": "https://betelgeuse.example.com/education_credential",
+  "name": "Betelgeuse Education Credential - Preliminary Version",
+  "description": "This is our development version of the education credential. Don't panic.",
+  "extends": "https://galaxy.example.com/galactic-education-credential-0.9",
+  "extends#integrity": "sha256-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-WRL5",
+  "display": [
+    {
+      "lang": "en-US",
+      "name": "Betelgeuse Education Credential",
+      "description": "An education credential for all carbon-based life forms on Betelgeusians",
+      "rendering": {
+        "simple": {
+          "logo": {
+            "uri": "https://betelgeuse.example.com/public/education-logo.png",
+            "uri#integrity": "sha256-LmXfh-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1V",
+            "alt_text": "Betelgeuse Ministry of Education logo"
+          },
+          "background_color": "#12107c",
+          "text_color": "#FFFFFF"
+        },
+        "svg_templates": [
+          {
+            "uri": "https://betelgeuse.example.com/public/credential-english.svg",
+            "uri#integrity": "sha256-8cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-9c",
+            "properties": {
+              "orientation": "landscape",
+              "color_scheme": "light",
+              "contrast": "high"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "lang": "de-DE",
+      "name": "Betelgeuse-Bildungsnachweis",
+      "rendering": {
+        "simple": {
+          "logo": {
+            "uri": "https://betelgeuse.example.com/public/education-logo-de.png",
+            "uri#integrity": "sha256-LmXfh-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1V",
+            "alt_text": "Logo des Betelgeusischen Bildungsministeriums"
+          },
+          "background_color": "#12107c",
+          "text_color": "#FFFFFF"
+        },
+        "svg_templates": [
+          {
+            "uri": "https://betelgeuse.example.com/public/credential-german.svg",
+            "uri#integrity": "sha256-8cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-9c",
+            "properties": {
+              "orientation": "landscape",
+              "color_scheme": "light",
+              "contrast": "high"
+            }
+          }
+        ]
+      }
+    }
+  ],
+  "claims": [
+    {
+      "path": [
+        "name"
+      ],
```

Comment:
I know it's not consistent with the other JSON formatting in this example but I think showing path this way conveys the idea much more intuitively.
```suggestion
      "path": ["name"],
```